client/asset/interface.go

```diff
@@ -215,17 +215,17 @@ type Receipt interface {
 
 // AuditInfo is audit information about a swap contract needed to audit the
 // contract.
-type AuditInfo interface {
+type AuditInfo struct {
 	// Recipient is the string-encoded recipient address.
-	Recipient() string
+	Recipient string
 	// Expiration is the unix timestamp of the contract time lock expiration.
-	Expiration() time.Time
+	Expiration time.Time
 	// Coin is the coin that contains the contract.
-	Coin() Coin
+	Coin Coin
 	// Contract is the contract script.
-	Contract() dex.Bytes
+	Contract dex.Bytes
 	// SecretHash is the contract's secret hash.
-	SecretHash() dex.Bytes
+	SecretHash dex.Bytes
 }
```

Comment:
I needed to modify the `Recipient` outside of **client/asset/btc**, so making this a field was a solution, but I didn't see any reason that any of these needed to be methods.